doc/particles.md

```diff
@@ -96,110 +99,110 @@ game.add(
+Simply translates underlying `Particle` to a specified `Vector2` within the rendering `Canvas`.
 Does not change or alter its position, consider using `MovingParticle` or `AcceleratedParticle` where change of position is required.
 Same effect could be achieved by translating the `Canvas` layer.
 
 ```dart
 game.add(
     ParticleComponent(
         particle: TranslatedParticle(
-            // Will translate child Particle effect to
-            // the center of game canvas
-            offset: game.size.center(Offset.zero),
+            // Will translate child Particle effect to the center of game canvas.
+            offset: game.size / 2,
             child: Particle(),
-        )
-    )
+        ),
+    ),
 );
 ```
 
 ## Moving Particle
 
-Moves child `Particle` between `from` and `to` `Offset`s during its lifespan. Supports `Curve` via `CurvedParticle`.
+Moves child `Particle` between `from` and `to` `Vector2`s during its lifespan. Supports `Curve` via `CurvedParticle`.
 
 ```dart
 game.add(
     ParticleComponent(
         particle: MovingParticle(
-            // Will move from corner to corner of the
-            // game canvas
-            from: game.size.topLeft(Offset.zero),
-            to: game.size.bottomRight(Offset.zero),
+            // Will move from corner to corner of the game canvas.
+            from: Vector2.zero(),
+            to: game.size,
             child: Particle(),
-        )
-    )
+        ),
+    ),
 );
 ```
 
 ## Accelerated Particle
 
-A basic physics particle which allows you to specify its initial `position`, `speed` and `acceleration` and let the `update` cycle do the rest. All three are specified as `Offset`s, 
+A basic physics particle which allows you to specify its initial `position`, `speed` and `acceleration` and let the `update` cycle do the rest. All three are specified as `Vector2`s, 
 which you can think of as vectors. It works especially well for physics-based "bursts", but it is not limited to that.
-Unit of the `Offset` value is _logical px/s_. So a speed of `Offset(0, 100)` will move a child `Particle` by 100 logical pixels of the device every second of game time.
+Unit of the `Vector2` value is _logical px/s_. So a speed of `Vector2(0, 100)` will move a child `Particle` by 100 logical pixels of the device every second of game time.
 
 ```dart
 final rnd = Random();
 game.add(
     ParticleComponent(
         particle: AcceleratedParticle(
             // Will fire off in the center of game canvas
-            position: game.size.center(Offset.zero),
-            // With random initial speed of Offset(-100..100, 0..-100)
-            speed: Offset(rnd.nextDouble() * 200 - 100, -rnd.nextDouble() * 100),
+            position: game.size.center(Vector2.zero()),
+            // With random initial speed of Vector2(-100..100, 0..-100)
+            speed: Vector2(rnd.nextDouble() * 200 - 100, -rnd.nextDouble() * 100),
```

Comment:
Could this use the same random function as above?
We really should add a Vector2 random extension.

Reply:
The random function is not even used it seems.